packages/edit-site/src/components/site-hub/index.js

```diff
@@ -32,7 +32,7 @@ import { unlock } from '../../lock-unlock';
 
 const HUB_ANIMATION_DURATION = 0.3;
 
-const SiteHub = forwardRef( ( { isTransparent, ...restProps }, ref ) => {
+const SiteHub = memo( ( { isTransparent, ...restProps } ) => {
```

Comment:
Now that you removed `variants` props. I guess we can just use `isTransparent` and `className` props. WDYT?

Reply:
Thanks, Riad. I think with clear list of props it would be "harder" to break memoization for a component.